### ISSUE TITLE
I'd like to see a reduce learning rate button or something.

### ISSUE THREAD

**Issue by mcourteaux** [NVlabs/instant-ngp#83]
For a few datasets, it seems nice to reduce the learning rate. I know the selling point of this paper is that it's fast, but to get some more detail in, I'd like to experiment with decreasing learning rate. I know it's in the config file, but would be cool to have a button to reduce by 50% or something.

**Comment by Tom94** [comment]
I'm hesitant to add case-specific features like this one to the GUI in the main repository. These sorts of features are more elegantly kept in separate branches.

Rather, for general use, I recommend adding a python binding to set the learning rate, similar to https://github.com/NVlabs/instant-ngp/issues/84 . The Python binding would only need to call `Testbed::m_optimizer->set_learning_rate(float)`.
